admin_manual/installation/example_openbsd.rst

```diff
@@ -0,0 +1,240 @@
+		  block return 301 "$DOCUMENT_URI/index.php"
+	  }
+
+	  location "/.well-known/carddav" {
+		  block return 301 "https://$SERVER_NAME/nextcloud/remote.php/dav"
+	  }
+
+	  location "/.well-known/caldav" {
+		  block return 301 "https://$SERVER_NAME/nextcloud/remote.php/dav"
+	  }
+
+	  location "/.well-known/webfinger" {
+		  block return 301 "https://$SERVER_NAME/nextcloud/public.php?service=webfinger"
+	  }
+
+	  location match "/nextcloud/oc[ms]%-provider/*" {
+		  directory index index.php
+		  pass
+	  }
+  }
+
+::
+
+Make sure that httpd(8) is enabled and started::
+
+    # rcctl enable httpd
+    # rcctl start httpd
+
+PHP
+---
+
+Assuming that you are on OpenBSD -current (or >= 6.8-stable) you could use PHP 7.4 so I will keep this version, but the concept is the same for other version.
+
+The PHP packages will be available since you installed Nextcloud with pkg_add, so you just need to adjust a bit your php.ini.
+
+It is recommended to add opcache to it::
+
+  [opcache]
+  opcache.enable=1
+  opcache.enable_cli=1
+  opcache.memory_consumption=512
+  opcache.interned_strings_buffer=8
+  opcache.max_accelerated_files=10000
+  opcache.revalidate_freq=1
+  opcache.save_comments=1
+::
+
+And increase some limits::
+
+  post_max_size = 513M
+  upload_max_filesize = 513M
+::
+
+   
+We can enable the PHP modules with::
+
+    # cd /etc/php-7.4.sample
+    # for i in *; do ln -sf ../php-7.4.sample/$i ../php-7.4/; done
+    
+And then we just enable and start PHP::
+
+    # rcctl enable php74_fpm
+    # rcctl start php74_fpm
+
+
+Database
+--------
+
+As mentioned, we will be using PostgreSQL as our database, and we already installed it, now we need to initialised::
+    
+    $ su - _postgresql
+    $ mkdir /var/postgresql/data
+    $ initdb -D /var/postgresql/data -U postgres -A md5 -E UTF8 -W
+    ...
+    Enter new superuser password: PASSWORD
+    Enter it again: PASSWORD
+    ...
+    Success. You can now start the database server using:
+
+    pg_ctl -D /var/postgresql/data -l logfile start
+
+    $ pg_ctl -D /var/postgresql/data -l logfile start
+    server starting
+    $ exit
+
+
+We need to check, enable and start postgres::
+
+    # rcctl check postgresql
+    # rcctl enable postgresql
+    # rcctl start postgresql
+    
+You can follow the README on ``/usr/local/share/doc/pkg-readmes/postgresql-server`` to create users and permission.
+
+
+Redis
+-----
+
+We installed redis before, we need to enable it and start it and also add it to the Nextcloud conf::
+
+    # rcctl enable redis
+    # rcctl start redis
+    # mg /var/www/nextcloud/config/config.php
+    ...
+      'memcache.local' => '\OC\Memcache\Redis',
+      'redis' => array(
+      'host' => 'localhost',
+      'port' => 6379,
+      'timeout' => 0.0,
+    ),
+    ...
+::
+
+Cron job
+--------
+
+We need to add the Nextcloud cron job to get some tasks done by adding this entry on your cronjob::
+
+  */15 * * * * /usr/bin/ftp -Vo - https://domain.tld/cron.php >/dev/null
```

Comment:
```suggestion
  */5 * * * * /usr/bin/ftp -Vo - https://domain.tld/cron.php >/dev/null
```

We also suggest to run this from CLI.

Reply:
Yes, but usually httpd(8) runs under a chroot() so for some users is more useful to just put a cronjob.